dask_ml/preprocessing/data.py

```diff
@@ -210,3 +210,22 @@ def _transform_col(self, X_col, quantiles, inverse):
             X_col = da.clip(X_col, clip_min, clip_max)
 
         return X_col
+
+
+class RobustScaler(skdata.RobustScaler):
+
+    def _check_array(self, X, copy):
+        return X
+
+    def fit(self, X, y=None):
+        q_min, q_max = self.quantile_range
+        if not 0 <= q_min <= q_max <= 100:
+            raise ValueError("Invalid quantile range: %s" %
+                             str(self.quantile_range))
+
+        quantiles = [da.percentile(col, [q_min, 50., q_max]) for col in X.T]
```

Comment:
It would be great to add the `axis` kwarg to `da.percentile` to reflect what is available in `np.percentile`.

BTW, I think the code `RobustScaler` in scikit-learn should also use `np.percentile` to compute both the median and the interquantile range at the same time as done here.